### ISSUE TITLE
captured ships shouldn't kamikaze 

### ISSUE THREAD

**Issue by johndh** [endless-sky/endless-sky#7774]
### Problem Description

One of the frustrating parts about capturing is that captured ships will (in their nearly dead state) immediately charge into combat and get destroyed, killing all of those valuable crew members along with the ship.
I'm anticipating that the counter-argument will be "anything that makes capping less difficult is bad", to which I reply that there's a difference between difficulty based on challenging gameplay, and difficulty based on frustrating/inconvenient gameplay. The former is good and the latter is bad. Boarding should be difficult in challenging ways, not frustrating ways, to offset the large rewards.

### Related Issue Links

#5494 and a million others

### Desired Solution

Newly captured ships can't fight at all until they land on a planet with an outfitter and pay some kind of repair fee for damaged equipment. Until then, they're in limp mode. After all, there are going to be blast holes in the equipment, crew are going to have to find their stations, they might be locked out of certain systems, etc. This would make capturing less frustrating because it increases survivability, but also be used to balance the reward.

### Alternative Approaches

Heavily-damaged ships could be more timid in general

**Comment by bene-dictator** [comment]
Yeah, this is a problem I've seen myself. At the moment it's possible to keep some of them out of the action if you get them to hold in place with "H" (as long as they're not that far away from enemy ships), but boarding and everything about it should be changed at some point to be more interesting and more believable than whatever lottery we have now.

**Comment by ThrawnCA** [comment]
I've generally addressed it by looting all their weapons before capturing.

**Comment by vitalchip** [comment]
I usually just right click on the radar to send them away and hold position. If you click near the circumference it'll send them about 10,000px away and provided you don't pick a really bad direction they'll be pretty safe.

**Comment by binaryspica** [comment]
Else it is possible that simply make a disabled ship to not use any weapons at all until it gets to the spaceport for full repair, although it makes the game complicated.

Anyway this issue is a really annoying one. I should say for this earlier.

**Comment by yafflehk** [comment]
It would make sense to have a general "Protect this ship" command for escort missions etc. Sort of the opposite of "F".

**Comment by brendanjones** [comment]
Definitely an annoying and unrealistic behaviour.

Wouldn't this be partially solved by the very simple change of captured ships defaulting to 'gather around flagship' behaviour until ordered otherwise? This way they don't race off to fight as soon as you exit the boarding screen, and stay under the protection of your flagship.
